scripts/install_plugins.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a utility script to install a plugin in the RAVEN plugin directory
```

Comment:
This looks like it does much more thorough testing than the other scripts, but what is the difference between all three?  Expanding the commented text might help clarify this.

Reply:
improved docstrings

Reply:
oh, this makes a lot more sense now.  Thanks.